src/simple-content-moderation/plugin.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Simple content moderation plugin that prevents saving annotations containing hate speech
+ *
+ * This plugin monitors text entered into TextArea regions and checks for the word "hate"
+ * before allowing the annotation to be saved. If found, it shows an error message and
+ * prevents submission.
+ *
```

Comment:
A comment about behaviour of this warning
```suggestion
 * This plugin monitors text entered into TextArea regions and checks for the word "hate"
 * before allowing the annotation to be saved. If found, it shows a warning message and
 * prevents submission. This would happen only once, if user clicks Submit again it would
 * work with no errors.
 *
```